bestoon/settings.py

```diff
@@ -114,8 +113,11 @@
 
 USE_TZ = True
 
-
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+# Secrets
+RECAPTCHA_SECRET_KEY = os.environ.get('RECAPTCHA_SECRET_KEY')
```

Comment:
@arashrasoulzadeh i think it's a convention for environment variables to be UPPERCASE 
 i suggest 
```
+RECAPTCHA_SECRET_KEY = os.environ.get('BESTOON_RECAPTCHA_SECRET_KEY')
```